runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20230921639352_changelog.xml

```diff
@@ -0,0 +1,37 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="gasiorowskaa" id="20230921639352-1">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <tableExists tableName="process"/>
+                <columnExists tableName="process" columnName="executions_count"/>
+                <columnExists tableName="process" columnName="success_executions_count"/>
+                <columnExists tableName="process" columnName="failure_executions_count"/>
+            </and>
+        </preConditions>
+
+        <dropColumn tableName="process" columnName="executions_count"/>
+        <dropColumn tableName="process" columnName="success_executions_count"/>
+        <dropColumn tableName="process" columnName="failure_executions_count"/>
+    </changeSet>
+
+    <changeSet author="gasiorowskaa" id="20230921639352-2">
+        <preConditions>
+            <tableExists tableName="process"/>
+            <not>
+                <columnExists tableName="process" columnName="last_run"/>
+                <columnExists tableName="process" columnName="editor_id"/>
+            </not>
+        </preConditions>
+
+        <addColumn tableName="process">
+            <column name="last_run" type="timestamp">
+                <constraints nullable="true"/>
+            </column>
+            <column name="editor_id" type="bigint">
+                <constraints nullable="false"/>
```

Comment:
imo nullable true here due to the possible deletion of the user

Reply:
here should be created foreign-key constraint for editor_id in another changeSet

Reply:
If there will be null, then user mapper will throw error 500. I've tried it and I wasn't sure if that needs to be changed.
 
I forgot about foreign key. I will add it.